test/board_test.exs

```diff
@@ -0,0 +1,43 @@
+defmodule BoardTest do
+  use ExUnit.Case
+
+  describe "board/0" do
+
+    test "populates initial board" do
+      assert Board.setup_initial_board() ===
+        %{1 => "1", 2 => "2", 3 => "3", 4 => "4", 5 => "5", 6 => "6", 7 => "7", 8 => "8", 9 => "9"}
+    end
+
+    test "converts the board into a CommandLine display friendly format" do
+      unformatted_board = Board.setup_initial_board()
```

Comment:
I'm generally not a big fan of having a test rely on code other than the one we're testing unless it's a test helper. I get that the idea here is to make for an easier test setup, but by doing so we're tying this tests to something other than the function we're testing here. For example, if we were to rename `setup_initial_board`, then all the tests that rely on this function would fell. I'm usually more tolerant on duplication when it come to tests and only use abstraction when I can do it without making the test brittle and difficult to reason about.

Reply:
Eli and i were actually just talking about this and this sounds like a good idea to me! Just made the change in the tests, thank you 😄